service/storage_proxy.cc

```diff
@@ -711,6 +711,18 @@ class storage_proxy::remote {
             co_return co_await encode_replica_exception_for_rpc<Result>(p->features(), f_s.get_exception());
         }
         schema_ptr s = f_s.get();
+
+        co_await utils::get_local_injector().inject_with_handler("storage_proxy::handle_read", [s] (auto& handler) -> future<> {
```

Comment:
Should we use more descriptive name in case we will have more than one injection point in the function? `storage_proxy::handle_read_pause`?

Reply:
No problem to rename this one if the need arises. I would prefer to keep it simple.